.php_cs.dist

```diff
@@ -1,6 +1,7 @@
 <?php
 
 $finder = PhpCsFixer\Finder::create()
+  ->exclude([__DIR__.'/tests/E2E/fixtures/mvcMatchBug'])
```

Comment:
though not mandatory, we do have a way to create test fixtures using a virtual file system, eg: https://github.com/phparkitect/arkitect/blob/master/tests/Unit/ClassSetTest.php#L69

This could avoid creating a lot of files or directories, just for testing purpose

Reply:
Yes, It's better to create test fixtures using a virtual file system. I agree with you. 
Thanks.

Reply:
We can't use a virtual file system because with integration tests it can't work at the moment.